shared/Dockerfile.test

```diff
@@ -5,7 +5,8 @@ RUN apt-get -y update && apt-get -y upgrade && \
 
 WORKDIR /app
 COPY requirements/ .
-RUN pip install -U -r requirements-testing.txt -r requirements.txt
+RUN touch requirements.txt
```

Comment:
why are you touching this file?

Reply:
Because `requirements-testing.txt` imports `requirements-general.txt` which imports `requirements.txt` and that fails as a whole if the file does not exist